music21/musicxml/m21ToXml.py

```diff
@@ -488,12 +488,26 @@ def fromMeasure(self, m):
         representation of a Measure, not for partial
         solutions in Part or Stream production.
         '''
+        m.coreGatherMissingSpanners()
         mCopy = m.makeNotation()
-        if not m.recurse().getElementsByClass('Clef').getElementsByOffset(0.0):
+        if mCopy.style.measureNumbering is None:
+            # Provide a default
+            mCopy.style.measureNumbering = 'measure'
```

Comment:
I think that measureNumbering = 'start' should probably be applied to Parts created with the .measures(start, end) tag.

Btw -- I have a lot of measureNumbering off because it wasn't imported in musicxml readers and no number is better than a wrong number.  I haven't checked recently (just got Finale 27) to see if this is till the case.

Reply:
Finale 27 properly imports measure numbers.